ush/python/pygfs/task/analysis.py

```diff
@@ -64,6 +68,49 @@ def get_obs_dict(self: Task) -> Dict[str, Any]:
         }
         return obs_dict
 
+    @logit(logger)
+    def get_bias_dict(self: Task) -> Dict[str, Any]:
+        """Compile a dictionary of observation files to copy
+
+        This method uses the OBS_LIST configuration variable to generate a dictionary
+        from a list of YAML files that specify what observation bias correction files
+        are to be copied to the run directory from the observation input directory
+
+        Parameters
+        ----------
+
+        Returns
+        ----------
+        bias_dict: Dict
+            a dictionary containing the list of observation bias files to copy for FileHandler
+        """
+        logger.debug(f"OBS_LIST: {self.task_config['OBS_LIST']}")
+        obs_list_config = parse_j2yaml(self.task_config["OBS_LIST"], self.task_config)
+        logger.debug(f"obs_list_config: {obs_list_config}")
+        # get observers from master dictionary
+        observers = obs_list_config['observers']
+        copylist = []
+        for ob in observers:
+            if 'obs bias' in ob.keys():
+                obfile = ob['obs bias']['input file']
+                basename = os.path.basename(obfile)
+                copylist.append([os.path.join(self.task_config.comin_ges_atm, basename), obfile])
+
+                obfile = obfile.replace('satbias', 'satbias_cov')
+                basename = os.path.basename(obfile)
+                copylist.append([os.path.join(self.task_config.comin_ges_atm, basename), obfile])
+
+                obfile = obfile.replace('satbias_cov', 'tlapse')
+                obfile = obfile.replace('nc4', 'txt')
+                basename = os.path.basename(obfile)
+                copylist.append([os.path.join(self.task_config.comin_ges_atm, basename), obfile])
```

Comment:
```suggestion
                obdir = os.path.dirname(obfile)

                for file in ['satbias.nc4', 'satbias_cov.nc4', 'lapse.txt']:
                    copylist.append([os.path.join(self.task_config.comin_ges_atm, file), os.path.join(obdir, file)])
```

Reply:
@aerorahul , I like the consolidation.   The suggested change, however, does not work.   The bias correction files begin with the prefix `gdas.t${cyc}z.`   For example, here's a satbias file in ROTDIR:
 `/scratch2/NCEPDEV/stmp1/Russ.Treadon/RUNDIRS/prufsatm/gdasatmanl_00/obs/gdas.t18z.amsua_n19.satbias.2021122118.nc4`.   

I can remove the 10-digit string (`2021122118`) from the observation yaml.   We want to retain the prefix, `gdas.t18z`.   The prefix needs to be included on the left hand side of copylist.

Reply:
Yeah I think I would suggest what I think you are proposing Russ:
```
gdas.t18z.amsua_n19.satbias.nc4
gdas.t18z.amsua_n19.satbias_cov.nc4
gdas.t18z.amsua_n19.lapse.txt
```
That should be fine right?

Reply:
I agree w/ dropping the 10-character string with date.  That seems superfluous.
You will need a little more even with that change.  I am working on it.

Reply:
@RussTreadon-NOAA Please try the updated suggestion.

Reply:
I will also take a look at this yaml file and see if there is a better way to loop over these files `satbias.nc4` `satbias_cov.nc4` and `lapse.txt`.
They must be part of the bigger yaml file.

Reply:
Thank you @aerorahul .  I tested a more verbose & less elegant way to form `prefix`.   This works if I manually rename the the bias correction files in my ROTDIR.   A proper test requires that I remove the GDATE string from the filename in the obs yaml and then cycle forward with this.  Let me do this tomorrow plus replace my `prefix` with your scripting.

Reply:
The following works
- adapt suggested changes in `analysis.py`
- remove `$GDATE` from bias correction filenames in observation yaml
- rename bias correcton files in `ROTDIR` without `$GDATE` in the filename

Question:  When we implement UFS-DA ATM in operations after GFS v17, what naming convention do we foresee being used in operations?    Will we 
- retain `$CDUMP.t${cyc}z.$file`
- include the 10-digit datetime string in the filename.  For example, `$CDUMP.${PDY}${cyc}.$file`

Whatever naming convention operations uses, the g-w `analysis.py` and GDASApp observation yamls need to be consistent with one another.   We have cross repo dependencies.

Reply:
Suggested change committed at ef5da493.